src/angular-app/bellows/core/offline/editor-data.service.ts

```diff
@@ -458,12 +458,12 @@ export class EditorDataService {
     // https://unicode.org/reports/tr44/#Diacritic
     // https://stackoverflow.com/a/37511463/10818013
 
-    return input.normalize('NFD').replace(/\p{Diacritic}/gu, '')
+    return input.normalize('NFD').replace(/\p{Diacritic}/gu, '').normalize('NFC');
```

Comment:
if changing the first `normalize('NFD')` to `'NFC'` would eliminate the need for your extra call here, I think it would be easier to understand "the why" if we come back to this code for any reason 6 months from now.  If it won't and we need both calls, I think we should either put a link to the discussion in a comment here as an answer to "why?" this code required both.

Reply:
We have to normalize to NFD first in order to get at the diacritic as a separate code point.  If you normalize to NFC first, then the `/\p{Diacritic}/` will never match anything (unless someone types a diacritic in isolation, which doesn't make sense).

Here is a working example from the JS console:
![image](https://user-images.githubusercontent.com/3444521/148861157-d2716555-d568-49b6-910b-f442cd5ab4a6.png)

And here is the character split up using Character Identifier:
![image](https://user-images.githubusercontent.com/3444521/148861234-4faa8c13-72eb-4c7e-984e-811eab312a4f.png)

`/\p{Diacritic}/` matches U+0308 because it has the Unicode property 'Diacritic'.  When in NFC form, the single character does NOT have the Unicode property 'Diacritic'.

Reply:
ok, thanks for explaining that nuance!  This stuff is a little crazy 🤪